Design/Instrument-2.0/ScanningInstruments_Design.md

```diff
@@ -18,6 +18,8 @@ Two classes related to this have already been partially implemented in Mantid, [
 
 Support for the moving instruments will be implemented via the `DetectorInfo` class. The usage `SpectrumInfo` class needs no direct knowledge of the time indexing. The methods on the `DetectorInfo` class for accessing detector information will be accessed by giving a detector index and an optional time index. It will also hold information on the start and end times for each time index. Any further information, for example to normalise to monitor counts, can be held in a time series sample log.
 
+Currently initialisation of workspaces is usually done in an incremental way. This will have performance impacts for the loader, but can also leave the workspace in an inconsistent state. For the scanning instrument design the workspace should be initialised all at once, via a `DetectorInfo` factory, which will ensure the loader creates a workspace in a consistent state.
```

Comment:
@ianbush Yes I agree. Could you be a bit more specific about what you mean about this for other readers (the first sentence).